labs/gps_lab/README.md

```diff
@@ -0,0 +1,68 @@
+#GPS LAB
+
+##Description
```

Comment:
Can you add more detail to the description of the lab - including the hardware, software, services used, programming languages, learning objectives etc. You can see an example here: https://github.com/microsoft/iot-curriculum/blob/main/labs/iot/environment-monitor/README.md

essentially copy the table and update it to match this lab.